test/common/test_io.cpp

```diff
@@ -315,6 +315,121 @@ TEST (PCL, concatenatePointCloud)
   }
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////
+TEST (PCL, concatenatePointCloud2)
+{
+  CloudXYZRGBA cloud_xyz_rgba;
+  cloud_xyz_rgba.push_back (pt_xyz_rgba);
+  cloud_xyz_rgba.push_back (pt_xyz_rgba);
+  cloud_xyz_rgba.push_back (pt_xyz_rgba);
+  cloud_xyz_rgba.push_back (pt_xyz_rgba);
+  cloud_xyz_rgba.push_back (pt_xyz_rgba);
+
+  CloudXYZRGBA cloud_xyz_rgba2;
+  cloud_xyz_rgba2.push_back (pt_xyz_rgba2);
+  cloud_xyz_rgba2.push_back (pt_xyz_rgba2);
+
+  pcl::PCLPointCloud2 cloud1, cloud2, cloud_out, cloud_out2, cloud_out3, cloud_out4;
+  pcl::toPCLPointCloud2 (cloud_xyz_rgba, cloud1);
+  pcl::toPCLPointCloud2 (cloud_xyz_rgba2, cloud2);
+
+  // Regular
+  EXPECT_TRUE (pcl::concatenate (cloud1, cloud2, cloud_out));
+
+  CloudXYZRGBA cloud_all;
+  pcl::fromPCLPointCloud2 (cloud_out, cloud_all);
+
+  EXPECT_EQ (cloud_all.size (), cloud_xyz_rgba.size () + cloud_xyz_rgba2.size ());
+  for (int i = 0; i < int (cloud_xyz_rgba.size ()); ++i)
+  {
+    EXPECT_XYZ_EQ (cloud_all[i], cloud_xyz_rgba[i]);
+    EXPECT_RGB_EQ (cloud_all[i], cloud_xyz_rgba[i]);
```

Comment:
We can use `EXPECT_RGBA_EQ` here; the line below is not needed then.

Reply:
Didn't make changes in test for `concatenatePointCloud`